### ISSUE TITLE
Missing coral record

### ISSUE THREAD

**Issue by kdelong1594** [nickmckay/LiPDverse#109]
Hi Nick, I could not find this record in the iso2k v.0_13_1 QC sheet
tonight. The paper is doi.org/10.1029/2008GL035634 and it is the Mayotte coral record by Zinke et al. 2008. It was in oceans 2k and in coralhydro2K, so I am sure it was in Iso2k as well since it is an older record and the data is on NOAA paleo https://www.ncdc.noaa.gov/paleo-search/study/8424. Not sure why I could not find it may site name, doi, or lat and long. Seems to be missing from the QC sheet.

**Comment by nickmckay** [comment]
Hmm. Indeed, this record was in PAGES2k - here is the lipdverse page. http://lipdverse.org/Ocn-Mayotte.Zinke.2008.html . However, I can't seem to find that it was ever in iso2k, it should be CO08ZIMA , right? @thompsod should we add this? Am I missing something?

**Comment by thompsod** [comment]
I agree-- this needs to be added into the database.  I think we did have it at some point, but unfortunately I am traveling and do not have my hard drive with our original/archived metadata files with me.  @kdelong1594 can you help fill in the metadata for this on the QC sheet once Nick creates it?  I am headed into the field, and not sure I will be able to help with this :/

**Comment by nickmckay** [comment]
Well @thompsod @kdelong1594 , it looks like this slipped through the cracks. Should I add it now?
